app/models/edition/custom_lead_image.rb

```diff
@@ -1,6 +1,8 @@
 module Edition::CustomLeadImage
   extend ActiveSupport::Concern
 
+  SVG_CONTENT_TYPE = "image/svg+xml".freeze
```

Comment:
Feels to me like this constant should live on the Image class, given it doesn't really apply to lead images?

Reply:
Yea i did think about popping it there, but it's currently only used in this context so i wasn't sure if it was good to keep it closer to the source type thing. Happy to move it over though.

Reply:
The other approach if you think it is makes sense to keep it on the lead image might be to change it to `INVALID_CONTENT_TYPES` or maybe allow list rather than ban list. But then you've got to deal with a set, which is mildly less ergonomic and potentially YAGNI. If we're not sure though then might as well leave it I would say, no point changing things for questionable benefit

Reply:
I've ended up moving it to the images model and adding a `sgv?` method